### ISSUE TITLE
Notebook Jumbotron needs some accessibility tweaks

### ISSUE THREAD

**Issue by manfromjupyter** [nbgallery/nbgallery#526]
- Icons in stats metadata have language icon as an aria-hidden but for whatever reason these got forgotten.
- `/` read as forward slash should not be in there
- Spaces are not appearing for all of the spans. Suggestion, the way we did it, we focused on rendering without stylesheet also but those breaks and spaces are not seen when using multiple elements. Suggestion to have 2 spans. One that is for screenreader and one that is strictly for rendering without stylesheet. Will look cleaner anyway. Still currently reads correctly to the user but that's only because we wrote numbers as numbers not in words so screenreader doesn't need space to know it's not part of the previous word.
- When editing tags, the "delete tag" icon is just read as "link times". Should say ex. "delete tag imported" and will inject "link" in there automatically. I surmise this also needs to be fixed on the upload modals and should check to make sure deleting search filters isn't the same.
- input field when tags are empty has no placeholder text

![link times instead of delete tag - with edit](https://user-images.githubusercontent.com/51969207/113184655-733fbe00-9223-11eb-8b09-cd50ee0e0127.png)
![icons not set to aria-hidden but should - with edit](https://user-images.githubusercontent.com/51969207/113184656-733fbe00-9223-11eb-9c84-6441c23f9abb.png)

**Comment by mcrutch** [comment]
You would have to attach to event "tokenfield:createdtoken" to then modify every one of the buttons in the token field to change it to have proper closing text, but the package we are using for tokens is no longer maintained, so it might be worth exploring alternatives if any are out there. 

Note: This affects ALL tokenfields, so it would have to account for any possible page this appears on, which is why it might be better to explore alternatives before going too deep.

**Comment by manfromjupyter** [comment]
Blocked by technology. Need to spend time to look at alternatives.
